Exploration Queries/InputEntity_Host/Host2Acc_PossibleSuccessfulBruteForce.txt

```diff
@@ -14,22 +14,22 @@ let SuccessfulLoginEventId = 4624;
 let FailedLoginEventId = 4625;
 let AccountsPossibleSuccessfulBruteForce = (v_Host_HostName:string, v_Host_NTDomain:string, v_Host_DnsDomain:string){
 SecurityEvent
-	| where AccountType == "User"
+	| where AccountType == 'User'
 		and tolower(Computer) startswith tolower(v_Host_HostName) 
 		and (
-			tolower(Computer) contains tolower(v_Host_NTDomain) or
-			tolower(Computer) endswith tolower(v_Host_DnsDomain)
+			(v_Host_NTDomain != '' and tolower(Computer) contains tolower(v_Host_NTDomain)) or
+			(v_Host_DnsDomain != '' andtolower(Computer) endswith tolower(v_Host_DnsDomain))
```

Comment:
Same here on isnotempty(v_Host_NTDomain) also there is no space between "and" and "tolower(Computer)"